src/screens/Episode/common/index.ts

```diff
@@ -0,0 +1,8 @@
+export type { PlayerEpisode } from './types'
```

Comment:
I'm not sure if we should create `index.ts` in `common` folder.

Reply:
I try removed this index, but without it grow amnt of imports. Constants and types we forced import diffrent lines in components.

Reply:
Yeah, I understand that you want to reduce amount of code. But we should create cusomizable, flexible code. Therefore we are using enums, types, contants etc. I think when we have separate imports such as `./common/constants`, `./common/types`, it's more clear for another developer

Reply:
> I try removed this index, but without it, grow amnt of imports. Constants and types we forced import diffrent lines in components.

Yeah, I understand that you want to reduce amount of code. But we should create cusomizable, flexible code. Therefore we are using enums, types, contants etc. I think when we have separate imports such as `./common/constants`, `./common/types`, it's more clear for another developer